src/timing.h

```diff
@@ -58,6 +58,7 @@ extern uint32_t jl_timing_print_limit;
 #if !defined( ENABLE_TIMINGS ) || !defined( HAVE_TIMING_SUPPORT )
 
 #define JL_TIMING(subsystem, event)
+#define JL_COND_TIMING(enabled, subsystem, event)
```

Comment:
What do you think about changing `JL_TIMING` to take the enable flag instead?

(third argument position is my vote)

Reply:
I would rather make it obvious that a particular profile event is conditional, rather than an overloaded macro. I think it's more readable overall.